models/multimeter.cpp

```diff
@@ -91,27 +98,55 @@ nest::Multimeter::Parameters_::set( const DictionaryDatum& d,
   const Buffers_& b )
 {
   if ( b.has_targets_
-    && ( d->known( names::interval ) || d->known( names::record_from ) ) )
+    && ( d->known( names::interval ) || d->known( names::offset )
+         || d->known( names::record_from ) ) )
+  {
     throw BadProperty(
-      "The recording interval and the list of properties to record "
-      "cannot be changed after the multimeter has been connected to "
-      "nodes." );
+      "The recording interval, the interval offset and the list of properties "
+      "to record "
+      "cannot be changed after the multimeter has been connected to nodes." );
+  }
 
   double v;
   if ( updateValue< double >( d, names::interval, v ) )
   {
     if ( Time( Time::ms( v ) ) < Time::get_resolution() )
+    {
       throw BadProperty(
-        "The sampling interval must be at least as long "
-        "as the simulation resolution." );
+        "The sampling interval must be at least as long as the simulation "
+        "resolution." );
+    }
 
     // see if we can represent interval as multiple of step
```

Comment:
Can you point out which lines are you referring to exactly? 121-129? It seems to me that it's just one way to get around the numerical issues, but the main idea is present in the comment

> // see if we can represent interval as multiple of step

Even if the interval is a multiple of step, `std::abs( 1 - interval_.get_ms() / v )` might be very close but still != 0, hence the comparison with epsilon. Should I expand on this?